File: WEEK 1/Python/Calculate_Miles_Without_Refueling.py
                    #Description:
            ################################
#Create a program that prompts the user for the capacity,
# in gallons, of an automobile's fuel tank and the miles
# per gallon (MPG) the automobile can be driven. 
#The program should output the number of miles the automobile,
# can be driven without refueling.

print("Enter the capacity of the automobile's fuel tank (in gallons):\n")
fuelCapacity = float(input()) 
print("Enter the miles per gallon (MPG) of the automobile:\n")
milesPerGallon = float(input()) 
milesWithoutRefueling = fuelCapacity * milesPerGallon
print(f"The automobile can be driven approximately {milesWithoutRefueling} miles without refueling.")

File: WEEK 1/C#/Calculate_Miles_Without_Refueling.cs

class Program {
    static void Main() {
        Console.Write("Enter fuel tank capacity (in gallons): ");
        double capacity = double.Parse(Console.ReadLine());

        Console.Write("Enter miles per gallon (MPG): ");
        double mpg = double.Parse(Console.ReadLine());

        double milesWithoutRefueling = capacity * mpg;
        Console.WriteLine($"The automobile can be driven {milesWithoutRefueling} miles without refueling.");
    }
}